examples/platforms/efr32/rail_config.h

```diff
@@ -0,0 +1,9 @@
+#ifndef __RAIL_CONFIG_H__
+#define __RAIL_CONFIG_H__
+
+#include <stdint.h>
+#include "rail_types.h"
+
+#define RADIO_CONFIG_XTAL_FREQUENCY 38400000UL
```

Comment:
Thanks for updating to RAIL 2.0 👍 . Please remove duplicated RADIO_CONFIG_XTAL_FREQUENCY and other parameters from openthread-core-efr32-config.h if we put auto-generated header file here.

Reply:
Thanks!  Updated.